Library/Homebrew/extend/predicable.rb

```diff
@@ -0,0 +1,9 @@
+module Predicable
+  def attr_predicate(*attrs)
+    attrs.each do |attr|
+      define_method attr do
+        instance_variable_get("@#{attr.to_s.sub(/\?$/, "")}") == true
```

Comment:
Consistency with Ruby is more important, IMO, but it looks like I'm in the minority.